vue-components/src/components/LookupMenu.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+	<div :class="[ 'wikit', 'wikit-LookupMenu' ]">
+		<div class="wikit-LookupMenu__item" :key="index" v-for="(menuItem, index) in menuItems">
+			<div class="wikit-LookupMenu__item__label">
+				{{ menuItem.label }}
+			</div>
+			<div class="wikit-LookupMenu__item__description">
+				{{ menuItem.description }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * This is an internal component which used by the Lookup component.
+ */
+export default Vue.extend( {
+	name: 'LookupMenu',
+	props: {
+		menuItems: {
+			type: Array,
+			default: (): [] => [],
+		},
+	},
+	watch: {
+		menuItems(): void {
+			const menuItems = this.$el.querySelectorAll( '.wikit-LookupMenu__item' );
+			if ( menuItems.length > 6 ) {
+				const menuHeight = menuItems[ 6 ].offsetTop - menuItems[ 0 ].offsetTop;
+				this.$el.style.maxHeight = menuHeight + 'px';
+			}
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+$base: '.wikit-LookupMenu';
+
+#{$base} {
+	min-width: 100%;
+	width: 200%;
```

Comment:
I am a bit confused by this, as these values are not encoded in design tokens, though I'm not sure if they should be?

Reply:
I had the same doubt while specifying the component. I guess that using alias tokens to define the width of the menu would be consistent, and convenient in case we make this a reusable element (comparable to the input).